client/asset/btc/walletclient.go

```diff
@@ -63,6 +63,7 @@ func (wc *walletClient) Balances() (*GetBalancesResult, error) {
 // ListUnspent retrieves a list of the wallet's UTXOs.
 func (wc *walletClient) ListUnspent() ([]*ListUnspentResult, error) {
 	unspents := make([]*ListUnspentResult, 0)
+	// TODO: listunspent 0 9999999 []string{}, include_unsafe=false
```

Comment:
This todo refers to the `include_unsafe=false` part of the RPC, which makes the dex app filtering redundant.  It's not terribly valuable though, just less for the app to worry about.